### DISCUSSION TITLE
Templater's function stopped working due to other plugins

### DISCUSSION THREAD

**Discussion by OofMon2301** [SilentVoid13/Templater#1021] [Help]
When using other community plugins (such as _Commander_ and _QuickAdd_), some functions stop working as intended. 

For example, function for setting the cursor to a location will not work and will output the code:
```javascript
 <% tp.file.cursor(1) %>
 ``` 

https://user-images.githubusercontent.com/114561394/216621940-dcc2f4d9-27e7-40a3-b0a2-41bdbe5f327b.mp4

Not sure how to fix it or what to do :<.
 
Continuation of #1013.
---
> What is that new lecture button? I dont have that in my vault o.o

It's a combination of _QuickAdd_ and _Commander_ (Both Community Plugins). _QuickAdd_ allows you to bind a specific command or macro to a hotkey, while _Commander_ allows you to be able to display it as a button on the status bar, ribbon, etc.

**Comment by welpdx** [comment]
Thank you for making this issue. Since I am not familiar with tp.file.cursor() [I did some testing](https://github.com/SilentVoid13/Templater/issues/928#issuecomment-1416926869). 

Can you take a look and tell me which method you are using that ultimately runs the templater? Thanks

**Reply by OofMon2301** [reply]
Hi, sorry for the late reply and inactivity.

Not exactly sure what you mean by method but I think you're referring to _QuickAdd_.
_QuickAdd_ just allows you to bind a number of actions/commands into a hotkey/command and can be accessed via Command Palette, etc.

This image shows the settings for _QuickAdd_:
![image](https://user-images.githubusercontent.com/114561394/218301976-652f909c-3558-48a9-abba-9bf4b9e401f3.png)

And this image shows the configuration within that macro:
<img src="https://user-images.githubusercontent.com/114561394/218301881-4be255c1-e709-48cf-8759-35e6e36ba7a1.png" width=500 />
